### PR TITLE
Hosting providers

### PR DESCRIPTION
**Pull request by berniedurfee**
Created a topic and initial standard for documenting permitted hosting providers.

### CODE REVIEW

**Comment by berniedurfee** [comment]
This will close #16

**Comment by tinomen** [comment]
So just had this convo with 2 teams last night. Perfect timing @berniedurfee 

I feel like we should also have a public facing docs or blog where students can contribute to "How To" articles for things like setting up hosts and what not.

File: docs/topics/hosting.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Hosting
+
+## (HT-100) Permitted Hosting Providers
+
+The following hosting providers are permitted for hosting application software:
+
+* Amazon Web Services (AWS)
+* Heroku
+* Zeit
+* Netlify
```
Comment (jessmartin) [inline docs/topics/hosting.md]: Why allow Netlify and Zeit? Why not just Heroku or AWS?

Comment (berniedurfee) [reply docs/topics/hosting.md]: I'd be fine eliminating Zeit and Netlify, keeping Heroku and AWS.

**Comment by berniedurfee** [comment]
This will close #13

**Pull request merged**